gateway-service/src/video/video.service.ts

```
@@ -0,0 +1,22 @@
+    @Inject('VIDEO_SERVICE') private readonly videoClient: ClientProxy,
+  ) {}
+  logger = new Logger('Gateway - Video Service');
+
+  async getCourseVideos(): Promise<any> {
+    try {
+      const result = await lastValueFrom(
+        this.videoClient.send('getCourseVideos', {}),
+      );
+      return result;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)